007_team_5_agent_design/007_team_5_agent_design.tex

```diff
@@ -1 +1,522 @@
+    \item It is badly formed. For example, a treaty which requires the agent to leave more than 100\% of the food on the platform is badly formed.
+    \item \ToDo{Check this one} It requires agents to leave less than a certain amount of food on the platform, as this is never advantageous to the tower as a whole.
+    \item \ToDo{Check this one} It requires the agent to leave an exact amount of food on the platform, as this would result in all but one of the agents signed up to the treaty to take no food each day.
+    \item It makes a request based on low HP, low amounts of food on the platform, or a low floor, as in these conditions the agent will already be desperate for survival, so will not want to be further constrained by a treaty.
+    \item \ToDo{Check this one} It makes a request based on HP where the threshold is within the ``weak'' or ``critical'' HP bands, as this means the agent could become critical or die as a result of following the treaty.
+    \item \ToDo{Check this one} It would require the agent to take less food than is required to avoid entering the ``critical'' HP band.
+\end{itemize}
+If the treaty does not meet any of the above conditions, the \texttt{decision} value is then computed as:
+\[\texttt{decision} = 6 - \texttt{selfishness} + \texttt{proposerFavour} - \frac{\texttt{treatyDuration}}{3}\]
+
+This calculation is based on the following considerations:
+\begin{itemize}
+    \item Selfish agents will always be less likely to sign treaties
+    \item The proposing agent is more likely to be trusted if the receiving agent views them more favourably
+    \item Longer treaties are viewed more negatively because factors such as the receiving agent's opinion of the proposer and the agent's selfishness can change over time, meaning the same treaty could be viewed less favourably in future. Therefore, the agent should avoid constraining its actions to satisfy an agent of which it could form a negative opinion.
+\end{itemize}
+
+If the \texttt{decision} value is positive, the treaty is accepted.
+
+\subsection*{Proposing Treaties}\label{sec:team5-treaties-proposal}
+At the start of each day, the agent checks whether it has become a `leader': if it has, then the agent proposes a treaty. Whether an agent becomes a leader or not is based on:
+\begin{itemize}
+    \item The agent's selfishness: a selfish agent is less likely to propose treaties
+    \item The agent's floor: agents higher up in the tower are more likely to propose treaties. This is because agents higher up in the tower have more access to food and therefore have more control over the food available to those below them. This control means these agents are in a better position to effect change.
```

Comment:
```suggestion
    \item The agent's floor: agents higher up in the tower are more likely to propose treaties. This is because an agents floor is the only indicator of social hierarchy within the tower, as those at the top have priority on the amount of food the eat and therefore control the amount of food available to agents below them. This control means these agents are in a better position to effect change.
```